src/planscape/forsys/write_forsys_output_to_db.py

```diff
@@ -10,12 +10,26 @@
 from pytz import timezone
 
 
+# Given a WKT, returns a MultiPolygon.
+def _get_multipolygon(wkt: str):
```

Comment:
@strangeties we have a function in plans/views.py _convert_polygon_to_multipolygon that works a bit differently. can we consolidate the two?

Reply:
sure - I've moved the common logic into plan/geos_utils.py